sentry-opentelemetry/sentry-opentelemetry-core/src/main/java/io/sentry/opentelemetry/SpanNode.java

```diff
@@ -0,0 +1,56 @@
+package io.sentry.opentelemetry;
+
+import io.opentelemetry.sdk.trace.data.SpanData;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SpanNode {
+  private final @NotNull String id;
+
+  // TODO [POTEL] should this be ReadableSpan? if so weak or strong ref?
+  private @Nullable SpanData span;
```

Comment:
Do we ever need to store a `ReadableSpan` for information that is not stored in a `SpanData`? 
If we get a `ReadableSpan` as input from somewhere we could call `ReadableSpan.toSpanData`

Reply:
Let's revisit this when implementation is further along. Currently we may not need `ReadableSpan`.

Reply:
Doesn't look like we need it so far, so I removed the TODO.